themes/default/content/migrate/arm2pulumi.md

```diff
@@ -7,9 +7,9 @@ menu:
   converters:
     identifier: arm2pulumi
     weight: 1
-    
+aliases:
+    - /migrate/arm2pulumi
```

Comment:
This is an attempt to get us some grouped formatting in Google search results (such that each conversion tool would appear underneath the overall Migrate to Pulumi heading). 🤞